recorder.go

```diff
@@ -244,6 +244,23 @@ func GetLightStepAccessToken(lsTracer ot.Tracer) (string, error) {
 	}
 }
 
+type Closer interface {
+	Close() error
+}
+
+func CloseTracer(tracer ot.Tracer) error {
```

Comment:
BTW this is only intended as a convenience function for the lightstep tracer. `Closer`, `basictracer`, etc are all incidental internal details. In the case of your custom Tracer interface, you could defer to using this to stop the lightstep tracer inside of it's Close method. We'll be removing the basictracer/recorder wrinkles soon (https://github.com/lightstep/lightstep-tracer-go/issues/86), so in the future you may be able to apply your TraceCloser interface directly to the lightstep tracer without a wrapper.